lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -15,6 +15,7 @@ $diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -d
 
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -failifempty $true
 $tags = Get-AnsibleParam -obj $params -name "tags" -type "list"
+$parameterlist = Get-AnsibleParam -obj $params -name "parameterlist" -type "dict"
```

Comment:
I'd call the new arg something like `test_parameters` since it's not a list, and we usually underscore multi-word names

Reply:
make sense, will change this